server/src/main/scala/org/scassandra/server/priming/json/ResultJsonRepresentation.scala

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
yep - I have also been meaning to move all the common things between PrimingJsonImplicits and the one for test into a separate class to remove the duplication.